src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -14,6 +14,24 @@
   ~    limitations under the License.
   -->
 <div class="p-3">
+  <mat-card
+    class="alert alert-info"
+    *ngIf="
+      !isPublic &&
+      workflow?.source_control_provider === 'GITHUB' &&
+      (workflow?.mode === WorkflowType.ModeEnum.FULL || workflow?.mode === WorkflowType.ModeEnum.STUB)
+    "
+  >
+    Keep your workflow automatically in sync with GitHub with our new registration process. Click
+    <a
+      href="{{ Dockstore.DOCUMENTATION_URL }}/faq.html#why-should-i-migrate-my-existing-workflow-to-use-github-apps-and-a-dockstore-yml"
```

Comment:
Seems like a big enough page that it should get its own page, but this can be changed simply enough later :man_shrugging:

Reply:
Yeah, that makes sense as it can be pretty important. Would that go in the getting started guide or advanced developer topics?

Reply:
Getting started , ideally we want this to (eventually?) be easy enough for all developers